cg/meta/workflow/rnafusion.py

```diff
@@ -197,9 +166,10 @@ def ensure_mandatory_metrics_present(metrics: list[MetricsBase]) -> None:
     def write_metrics_deliverables(self, case_id: str, dry_run: bool = False) -> None:
         """Write <case>_metrics_deliverables.yaml file."""
         metrics_deliverables_path: Path = self.get_metrics_deliverables_path(case_id=case_id)
-        metrics = self.get_multiqc_json_metrics(case_id=case_id)
+        metrics = self.get_multiqc_json_metrics(
```

Comment:
Related to the previous comment, if we remove the pipeline metrics from here then in the future it will be also easie to reuse the `write_metrics_deliverables` function

Reply:
Adressed in the comment above.